apps/vr-tests-react-components/src/stories/Button/Button.stories.tsx

```diff
@@ -21,27 +21,27 @@ export const DefaultHighContrast = getStoryVariant(Default, HIGH_CONTRAST);
 
```

Comment:
## [🕵 fluentuiV9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28261&commitId=2269d145ed6b8d72fa7d3b352ad30c78197b84fd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Converged</b> 84 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.As an anchor.default.chromium.png|123| Changed|
|Button Converged.As an anchor.hover.chromium.png|128| Changed|
|Button Converged.Circular.hover.chromium.png|60| Changed|
|Button Converged.Circular.default.chromium.png|58| Changed|
|Button Converged.Circular.pressed.chromium.png|112| Changed|
|Button Converged.Disabled - High Contrast.default.chromium.png|86| Changed|
|Button Converged.Disabled - High Contrast.hover.chromium.png|86| Changed|
|Button Converged.Disabled.hover.chromium.png|38| Changed|
|Button Converged.Disabled - Dark Mode.hover.chromium.png|51| Changed|
|Button Converged.As an anchor.pressed.chromium.png|236| Changed|
|Button Converged.Disabled - Dark Mode.pressed.chromium.png|51| Changed|
|Button Converged.Outline Disabled - Dark Mode.hover.chromium.png|14| Changed|
|Button Converged.Disabled - High Contrast.pressed.chromium.png|86| Changed|
|Button Converged.Outline Disabled - Dark Mode.pressed.chromium.png|14| Changed|
|Button Converged.Disabled - Dark Mode.default.chromium.png|51| Changed|
|Button Converged.Disabled.pressed.chromium.png|38| Changed|
|Button Converged.Disabled.default.chromium.png|38| Changed|
|Button Converged.Outline Disabled - Dark Mode.default.chromium.png|14| Changed|
|Button Converged.Outline Disabled - High Contrast.default.chromium.png|77| Changed|
|Button Converged.Outline Disabled - High Contrast.pressed.chromium.png|77| Changed|
|Button Converged.Outline Disabled - High Contrast.hover.chromium.png|77| Changed|
|Button Converged.Outline Disabled.default.chromium.png|18| Changed|
|Button Converged.Outline Disabled.hover.chromium.png|18| Changed|
|Button Converged.Outline Disabled.pressed.chromium.png|18| Changed|
|Button Converged.Outline.default.chromium.png|49| Changed|
|Button Converged.Outline.hover.chromium.png|53| Changed|
|Button Converged.Primary - Dark Mode.pressed.chromium.png|39| Changed|
|Button Converged.Primary - High Contrast.default.chromium.png|417| Changed|
|Button Converged.Primary - High Contrast.pressed.chromium.png|286| Changed|
|Button Converged.Outline.pressed.chromium.png|59| Changed|
|Button Converged.Primary - Dark Mode.hover.chromium.png|95| Changed|
|Button Converged.Primary Disabled - Dark Mode.hover.chromium.png|31| Changed|
|Button Converged.Primary - High Contrast.hover.chromium.png|286| Changed|
|Button Converged.Primary - Dark Mode.default.chromium.png|80| Changed|
|Button Converged.Primary Disabled - Dark Mode.default.chromium.png|31| Changed|
|Button Converged.Primary Disabled - High Contrast.hover.chromium.png|17| Changed|
|Button Converged.Primary Disabled - Dark Mode.pressed.chromium.png|31| Changed|
|Button Converged.Primary Disabled - High Contrast.pressed.chromium.png|17| Changed|
|Button Converged.Primary Disabled - High Contrast.default.chromium.png|17| Changed|
|Button Converged.Primary Disabled.default.chromium.png|22| Changed|
|Button Converged.Primary Disabled.pressed.chromium.png|22| Changed|
|Button Converged.Primary.pressed.chromium.png|337| Changed|
|Button Converged.Primary.default.chromium.png|270| Changed|
|Button Converged.Primary.hover.chromium.png|288| Changed|
|Button Converged.Primary Disabled.hover.chromium.png|22| Changed|
|Button Converged.Size small.hover.chromium.png|19| Changed|
|Button Converged.Size small.default.chromium.png|17| Changed|
|Button Converged.Size small.pressed.chromium.png|33| Changed|
|Button Converged.Subtle - High Contrast.default.chromium.png|42| Changed|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|35| Changed|
|Button Converged.Subtle - Dark Mode.default.chromium.png|29| Changed|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|57| Changed|
|Button Converged.Subtle - High Contrast.hover.chromium.png|313| Changed|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|313| Changed|
|Button Converged.Subtle Disabled - Dark Mode.pressed.chromium.png|9| Changed|
|Button Converged.Subtle Disabled - High Contrast.hover.chromium.png|31| Changed|
|Button Converged.Subtle Disabled - Dark Mode.default.chromium.png|9| Changed|
|Button Converged.Subtle Disabled.default.chromium.png|12| Changed|
|Button Converged.Subtle Disabled - High Contrast.default.chromium.png|31| Changed|
|Button Converged.Subtle.default.chromium.png|31| Changed|
|Button Converged.Subtle Disabled - Dark Mode.hover.chromium.png|9| Changed|
|Button Converged.Subtle.hover.chromium.png|35| Changed|
|Button Converged.Subtle Disabled - High Contrast.pressed.chromium.png|31| Changed|
|Button Converged.Subtle.pressed.chromium.png|81| Changed|
|Button Converged.Subtle Disabled.hover.chromium.png|12| Changed|
|Button Converged.Transparent - Dark Mode.hover.chromium.png|11| Changed|
|Button Converged.Subtle Disabled.pressed.chromium.png|12| Changed|
|Button Converged.Transparent - High Contrast.default.chromium.png|27| Changed|
|Button Converged.Transparent - High Contrast.hover.chromium.png|281| Changed|
|Button Converged.Transparent Disabled - Dark Mode.hover.chromium.png|7| Changed|
|Button Converged.Transparent - Dark Mode.default.chromium.png|18| Changed|
|Button Converged.Transparent Disabled - Dark Mode.default.chromium.png|7| Changed|
|Button Converged.Transparent - High Contrast.pressed.chromium.png|281| Changed|
|Button Converged.Transparent Disabled - Dark Mode.pressed.chromium.png|7| Changed|
|Button Converged.Transparent Disabled - High Contrast.default.chromium.png|26| Changed|
|Button Converged.Transparent - Dark Mode.pressed.chromium.png|8| Changed|
|Button Converged.Transparent Disabled - High Contrast.pressed.chromium.png|26| Changed|
|Button Converged.Transparent Disabled.hover.chromium.png|10| Changed|
|Button Converged.Transparent Disabled.pressed.chromium.png|10| Changed|
|Button Converged.Transparent.pressed.chromium.png|19| Changed|
|Button Converged.Transparent.hover.chromium.png|18| Changed|
|Button Converged.Transparent Disabled.default.chromium.png|10| Changed|
|Button Converged.Transparent Disabled - High Contrast.hover.chromium.png|26| Changed|
|Button Converged.Transparent.default.chromium.png|20| Changed|

</details>


  <details><summary><b>Field</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.Select.default.chromium.png|33986| Changed|
|Field.RadioGroup.default.chromium.png|55125| Changed|

</details>


  <details><summary><b>Radio Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.group-disabled.default.chromium.png|20399| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|20400| Changed|
|Radio Converged.group-vertical.default.chromium.png|20409| Changed|
|Radio Converged.group-vertical - RTL.default.chromium.png|20410| Changed|

</details>


  <details><summary><b>Select Converged</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Select Converged.Appearance- filled-darker.focused.chromium.png|7090| Changed|
|Select Converged.Appearance- outline (default).default.chromium.png|6519| Changed|
|Select Converged.Appearance- filled-darker.hover.chromium.png|6412| Changed|
|Select Converged.Appearance- underline.default.chromium.png|6518| Changed|
|Select Converged.Appearance- underline.hover.chromium.png|6518| Changed|
|Select Converged.Custom Icon.default.chromium.png|6534| Changed|
|Select Converged.Disabled.default.chromium.png|6409| Changed|
|Select Converged.Appearance- filled-darker.default.chromium.png|6412| Changed|
|Select Converged.Appearance- underline.focused.chromium.png|6502| Changed|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|7227| Changed|
|Select Converged.Invalid- filled-darker.focused.chromium.png|7090| Changed|
|Select Converged.Invalid- outline.focused.chromium.png|6503| Changed|
|Select Converged.Invalid- underline.default.chromium.png|6518| Changed|
|Select Converged.Invalid- outline.hover.chromium.png|6524| Changed|
|Select Converged.Appearance- filled-lighter.default.chromium.png|6528| Changed|
|Select Converged.Invalid- underline.focused.chromium.png|6502| Changed|
|Select Converged.Size- small.default.chromium.png|3329| Changed|
|Select Converged.Invalid- filled-darker.default.chromium.png|6738| Changed|
|Select Converged.Invalid- filled-lighter.default.chromium.png|6864| Changed|
|Select Converged.Invalid- underline.hover.chromium.png|6518| Changed|
|Select Converged.With appearance override.default.chromium.png|13766| Changed|
|Select Converged.With value.default.chromium.png|6594| Changed|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|7227| Changed|
|Select Converged.Size- large.default.chromium.png|6567| Changed|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|6864| Changed|
|Select Converged.Invalid- filled-darker.hover.chromium.png|6738| Changed|
|Select Converged.Appearance- outline (default).focused.chromium.png|6503| Changed|
|Select Converged.Invalid- outline.default.chromium.png|6524| Changed|
|Select Converged.Appearance- outline (default).hover.chromium.png|6520| Changed|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|6528| Changed|

</details>




<div id="vrtCommentE2E Tests-v9"/>